apps/extension/src/core/domains/sitesAuthorised/store.ts

```diff
@@ -39,8 +39,7 @@ export class SitesAuthorizedStore extends SubscribableByIdStorageProvider<
     if (err) throw new Error(val)
 
     const site = await this.get(val)
-
-    return { ...site, connectWatchedAccounts: isTalismanHostname(site.url) }
+    return { ...site, connectWatchedAccounts: isTalismanHostname(site?.url) }
```

Comment:
Weird, does type check not catch this?

Reply:
Yeah! The store returns `T` for `.get` 😞